internal/internal_workflow_testsuite.go

```diff
@@ -2249,7 +2249,7 @@ func (env *testWorkflowEnvironmentImpl) signalWorkflow(name string, input interf
 		panic(err)
 	}
 	env.postCallback(func() {
-		env.signalHandler(name, data)
+		_ = env.signalHandler(name, data)
```

Comment:
I'm curious, this seems like a JetBrains-ism, is it? I don't think any traditional go tools have any problem with ignoring these returns.

Reply:
We have errorcheck linter, which complains if you ignore an error.